substrate-processor/src/ingest.ts

```diff
@@ -7,13 +7,23 @@ import {AbortHandle, Channel, wait} from "./util/async"
 import {hasProperties, unique} from "./util/misc"
 import {rangeEnd} from "./util/range"
 
-
+/**
+ * Defines block data as part of the ingestion inflow
```

Comment:
What does it mean "as part of the ingestion flow"? Probably you mean that this is the data retrieved for each block?

Reply:
Pretty much, yeah. I put myself in a corner by starting with "block data", instead of "data that..."